File: pharmcrew-admin/src/main/java/kr/ant/kpa/pharmcrew/resp/ptax/Ptaxqna.java
package kr.ant.kpa.pharmcrew.resp.ptax;

import com.bumdori.spring.annotation.Description;
import kr.ant.kpa.pharmcrew.resp.common.AttachFile;
import lombok.Data;

import java.util.List;

@Data
public class Ptaxqna {
    private Long qnaId;
    private String memberName;
    private String pharmName;
    private String title;
    private String body;
    private String isAnswered;
    private String ansTitle;
    private String ansBody;
    private Long questionDt;
    private String adminName;
    private Long answerDt;
    /**
     * 문의 첨부 파일 리스트
     */
    @Description("문의 첨부 파일 리스트")
    private List<AttachFile> qAttachFiles;
    /**
     * 답변 첨부 파일 리스트
     */
    @Description("답변 첨부 파일 리스트")
    private List<AttachFile> aAttachFiles;
}

File: pharmcrew-admin-ng/src/app/domain/ptax/ptaxqna.ts
import { AttachFile } from '../common/attach-file';

export class Ptaxqna {
    qnaId: number;
    memberName: string;
    pharmName: string;
    title: string;
    body: string;
    isAnswered: string;
    ansTitle: string;
    ansBody: string;
    questionDt: number;
    adminName: string;
    answerDt: number;
    qAttachFiles: AttachFile[];
    aAttachFiles: AttachFile[];
}
